src/components/VmStatus/VmStatus.js

```diff
@@ -63,8 +68,13 @@ StateV2VConversionInProgress.propTypes = {
 };
 
 const StateV2VConversionError = ({ ...props }) => (
-  <LinkStatus icon="error-circle-o" {...props}>
-    V2V Conversion Error
+  <LinkStatus
+    icon="error-circle-o"
+    title="Importing (VMWare) Error"
+    information="Virtual Machine is being converted from VMware format to libvirt format."
```

Comment:
shouldn't there be some error message ?

Reply:
Can you add messages to other statuses ? Or at least create a followup issue

Reply:
> Can you add messages to other statuses ?

:+1: I just don't know what to write ...

@matthewcarleton help :-) 
I need help writing messages for the statuses, 
I do not what the message to just repeat the status, but I can't think of something usefull, can you help ?

Reply:
add some info for states:
@rawagner @matthewcarleton please review